_posts/2022-03-16-encodecall-bug.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+published: true
+title: 'abi.encodeCall Bug'
```

Comment:
That sounds like it happens whenever hex or string literals are encoded :-). If we move towards something more precise, it would also have to include ``fixed bytes types```:-). But to me also ``abi.encodeCall Bug`` seems fine...

Reply:
Generic titles are annoying. It's nice if it's specific enough that you can more or less tell what it's about when just skimming a list of bugs. The current one forces you to look inside to find out.

Reply:
Please suggest a short tittle and a different name for the file.